doc/source/report/behavior.rst

```diff
@@ -138,6 +138,18 @@ The data we require:
 - The above dataframe with a classification of consumption size and moving
   distance with each event at each time $t$
 
+A graphic view of the above is as below:
+
+======  =====  ======  =====  =======  ======
+strain  mouse   time   IS/AS  D/F/L/S  Consn.
+======  =====  ======  =====  =======  ======
+0       1      1.1     IS     S        0
+0       1      1.4     AS     F        4
+1       2      1.1     AS     D        5
+1       2      1.3     AS     D        3
+1       2      1.6     IS     S        0
+======  =====  ======  =====  =======  ======
+
```

Comment:
Can you clarify here what D/F/L/S is? I'm guessing it's Drink/Food/Locomotion/Stationary?